engineapi/engineapi/actions.py

```diff
@@ -1265,8 +1413,10 @@ def add_scores(
 
 
 def create_leaderboard_resource(
-    leaderboard_id: uuid.UUID,
-    token: Optional[uuid.UUID] = None,
+    leaderboard_id: str,
+    token: Union[Optional[uuid.UUID], str] = None,
```

Comment:
I think the correct typing should looks like:
`token: Optional[Union[uuid.UUID, str] = None`